src/ucs/config/global_opts.h

```diff
@@ -51,6 +51,8 @@ typedef struct {
     /* Enable FIFO behavior for memory pool, instead of LIFO. Useful for
      * debugging because object pointers are not recycled. */
     int                        mpool_fifo;
+    /* Maximum chunk size for memory pool */
+    size_t                     mpool_max_chunk_size;
```

Comment:
it should be parameter for each mem mpool: not all pools are allocated on shared memory

Reply:
added this parameter for each mpool